### DISCUSSION TITLE
Migrate from old version to latest on

### DISCUSSION THREAD

**Discussion by maximoolle** [stalwartlabs/jmap-server#63] [Q&A]
Hi
I installed docker version v0.3.7 and would like to upgrade to the latest one
I the wiki page, it's mentionned to export using $ stalwart-cli
Until now, I had never the occasion to use it 
Unfortunately, I cannot connect, getting the error when I am in the docker instance : 
`Failed to connect to JMAP server http://127.0.0.1:8080: Transport error: error sending request for url (http://127.0.0.1:8080/.well-known/jmap): invalid HTTP version parsed.
`
When connecting with navigator I got this strange output
`�2`

What would be the path to be able to proceed to my export and upgrade with the latest docker image ?

Thank you for the suggestion

**Comment by mdecimus** [comment]
Hi,

This looks like your server is configured with HTTPS and you are trying to connect over HTTP. Alternatively you might have the wrong URL configured on your listener.
Can you confirm whether port `8080` is plain text or TLS?

**Comment by maximoolle** [comment]
Sorry for the delay

Port 8080 is with TLS (am using traeffik)

**Comment by maximoolle** [comment]
Is there an alternate way to export my data before upgrading ?

**Reply by mdecimus** [reply]
Yes:

https://stalw.art/docs/management/cli/export/

**Reply by maximoolle** [reply]
Thanks for your feedback but this link points to the command I am unfortunately not able to run because of http error :/  
None of stalwart-cli commands is working

**Reply by mdecimus** [reply]
Try disabling traefik and connect directly to Stalwart.
